### ISSUE TITLE
Duplicate ScopeClosingBraceSniff

### ISSUE THREAD

**Issue by CarsonF** [djoos/Symfony-coding-standard#102]
`PSR2` imports `Squiz.WhiteSpace.ScopeClosingBrace`.
`Zend` imports `PEAR.WhiteSpace.ScopeClosingBrace`.

It's hard to tell which one is better. I'm leaning towards keeping the one in PSR over Zend though.

**Comment by wickedOne** [comment]
thanks for your input @CarsonF !

there are more duplicates (``ClassDeclaration.MissingBrace``, ``ScopeClosingBrace.indent`` to name a few)

it's not that this blocks phpcs from running nor that either of them is raising false positives.
in the end the violation needs to be fixed and after that both approve.

could you elaborate why we need to disable one of both?

**Comment by CarsonF** [comment]
Yes you are right. It just makes it a bit harder to sort through the list of errors in the report summary. I wouldn't have brought it up if it wasn't a simple one liner to disable the rule.

**Comment by wickedOne** [comment]
yes, feel free to create a PR (personally i'd keep the ``PSR2`` as well).
